migrations/script.py.mako

```diff
@@ -0,0 +1,16 @@
+from alembic import op
```

Comment:
why does everyone invent its own templating language?
`Mako's syntax and API borrows from the best ideas of many others, including Django and Jinja2 templates, Cheetah, Myghty, and Genshi.`
hopefully we touch this template very little

Reply:
Indeed.

Reply:
I was just thinking that looking at all that awful generated migration code. wtf.